### ISSUE TITLE
imageio.reader maintains connection to file even after it's closed

### ISSUE THREAD

**Issue by mattdornfeld** [imageio/imageio#201]
So my use case is mounting a remote folder, which contains several video files, with sshfs. Then making a connection to the video files with imageio.reader. When I close my program I want to close the video files and unmount the remote folder. However imageio.reader seems to be maintaining a connection to the file even after it's closed. Here is a minimal code example

```
import subprocess
import imageio
import os

mount_command = 'sshfs -o ro -o allow_other -o IdentityFile={} {}@{}:{} {}'.format(
		key_file, host_username, host_ip, host_data_directory, local_data_directory)
subprocess.call(mount_command, shell=True)
video_file = os.path.join(local_data_directory, file_name) 
video = imageio.get_reader(video_file,  'ffmpeg')
video.close()
unmount_command = 'fusermount -u {}'.format(local_data_directory)
subprocess.call(unmount_command, shell=True)
```

This gives the error

`fusermount: failed to unmount $local_data_directory: Device or resource busy
`
Is there a way to fully terminate the connection between reader and the video_file?

**Comment by almarklein** [comment]
It could be that the plugin (ffmpeg) does not close the file correctly. Some changes were recently made to the ffmpeg plugin that could relate to this. Did you try with the latest version (v2.1.x)? Also, just to make sure, do you only use the ffmpeg reader in between mounting and unmounting, so no other plugins can be the source of this bug?

**Comment by almarklein** [comment]
This is probbaly fixed with #213 (which is merged in the current master), could you please confirm?

**Comment by almarklein** [comment]
I am closing this as it is likely fixed now. Please reopen if it persists.
